File: Pheeder/Models/dbModel/DBFeedItems+CoreDataProperties.m
//
//  DBFeedItems+CoreDataProperties.m
//  Pheeder
//
//  Created by Deepu on 10/08/20.
//  Copyright © 2020 Deepak S. All rights reserved.
//
//

#import "DBFeedItems+CoreDataProperties.h"

@implementation DBFeedItems (CoreDataProperties)

+ (NSFetchRequest<DBFeedItems *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"DBFeedItems"];
}

@dynamic brief;
@dynamic content;
@dynamic date;
@dynamic isBookmared;
@dynamic tiitle;
@dynamic url;
@dynamic subscription;

@end

File: Pheeder/Models/dbModel/DBFeedItems+CoreDataProperties.swift
//
//  DBFeedItems+CoreDataProperties.swift
//  Pheeder
//
//  Created by deepak on 13/07/20.
//  Copyright © 2020 Deepak S. All rights reserved.
//
//

import Foundation
import CoreData
import Pheeder

extension DBFeedItems {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<DBFeedItems> {
        return NSFetchRequest<DBFeedItems>(entityName: "DBFeedItems")
    }

    @NSManaged public var brief: String?
    @NSManaged public var content: String?
    @NSManaged public var date: Date?
    @NSManaged public var isBookmared: Bool
    @NSManaged public var tiitle: String?
    @NSManaged public var url: URL?
    @NSManaged public var subscription: DBFeedSubscriptions?

}
